Sources/Internal/Version.swift

```diff
@@ -14,7 +14,68 @@
 
 import Foundation
 
+import GoogleUtilities_Environment
+
 @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
 struct Version {
-  static let version = "11.3.0-beta"
+  static let sdkVersion = "11.3.0-beta"
+
+  // returns value of form gl-PLATFORM_NAME/PLATFORM_VERSION
+  static func platformVersionHeader() -> String {
+    let versionString = GULAppEnvironmentUtil.systemVersion()
+    let platformName = GULAppEnvironmentUtil.applePlatform()
+
+    return "gl-\(platformName)/\(versionString)"
+  }
+
+  // returns the build time major version of swift
+  static func swiftVersion() -> String {
+    #if swift(>=6)
+      return "6"
+    #elseif swift(>=5.10)
+      return "5.10"
+    #elseif swift(>=5.9)
+      return "5.9"
+    #elseif swift(>=5.8)
```

Comment:
This is probably fine, but in practice nothing should ever come up below 5.9 since the minimum Xcode is 15.2

Reply:
True. I'll leave it in for merely for completeness.